test/src/main/java/se/fortnox/reactivewizard/test/WaitingTestUtils.java

```diff
@@ -0,0 +1,63 @@
+package se.fortnox.reactivewizard.test;
+
+import org.junit.Assert;
+import rx.Observable;
+import rx.Subscription;
+
+import java.util.concurrent.CountDownLatch;
+import java.util.concurrent.TimeUnit;
+import java.util.function.Supplier;
+
+import static java.util.concurrent.TimeUnit.MILLISECONDS;
+
+/**
+ * Small class for verifying things in a certain time.
+ */
+public abstract class WaitingTestUtils {
+
+    private static final String DEFAULT_ERROR_MESSAGE = "Condition not met within time";
+    private static final int DEFAULT_WAITING_SECONDS = 5;
+    private static final TimeUnit DEFAULT_WAITING_TIME_UNIT = TimeUnit.SECONDS;
+
+    private WaitingTestUtils() {}
+
+    public static void assertConditionIsTrueWithin5Seconds(Supplier<Boolean> condition) {
```

Comment:
assertConditionIsTrueWithinDefaultTimeout mabey, otherwise if we want to change the default timeout, these functions should change name, breaking tests.